### ISSUE TITLE
Seq.sumBy and Seq.sum don't resolve static plus members of custom types

### ISSUE THREAD

**Issue by Zaid-Ajaj** [fable-compiler/Fable#1582]
### Description
Seq.sumBy and Seq.sum don't resolve static plus members of custom types

### Repro code

```fs
type Number =  
    | Number of decimal

    static member (+) (Number(a), Number(b)) = Number(a + b)
    static member Zero = Number 0m

// works 
[ Number 5m; Number 10m  ]
|> Seq.fold (fun acc number -> acc + number) Number.Zero
|> printfn "%A"

// doesn't work
[ Number 5m; Number 10m  ]
|> Seq.sumBy id
|> printfn "%A"

// doesn't work
[ Number 5m; Number 10m  ]
|> Seq.sum
|> printfn "%A"
```

It seems to be a [known issue](https://github.com/fable-compiler/Fable/blob/master/tests/Main/SeqTests.fs#L391-L405)? 

### Related information

* Fable version (`dotnet fable --version`): REPL(2.0.0-beta-005)   
* Operating system: N/A

**Comment by alfonsogarciacaro** [comment]
True, that was a TODO that didn't make it into Fable 2.0.0, sorry. Probably something I can do on the plane back ;)

**Comment by alfonsogarciacaro** [comment]
Should be fixed in 2.0.2, can you please check and close if it works for you? Thank you!

**Comment by Zaid-Ajaj** [comment]
I don't have a repro project for now but I will trust the working tests :smile:
